testing/integration/linux_deb_test.go

```diff
@@ -176,6 +176,10 @@ func TestDebFleetUpgrade(t *testing.T) {
 	out, err := cmd.CombinedOutput() // #nosec G204 -- Need to pass in name of package
 	require.NoError(t, err, string(out))
 
+	cmd2 := exec.CommandContext(ctx, "sudo", "systemctl", "start", "elastic-agent")
+	out2, err := cmd2.CombinedOutput() // #nosec G204 -- Need to pass in name of package
+	require.NoError(t, err, string(out2))
+
```

Comment:
Why is this starting of elastic-agent needed here? I think it is started already as part of the install process for deb files...

Reply:
@leehinman knows better. This test was failing. I can remove it and try again

Reply:
Deb install did not start elastic-agent automatically.  I think @blakerouse fixed that.

Reply:
Thanks. Removing it